site/src/pages/[shortname]/blocks/[block-slug].page.tsx

```diff
@@ -165,9 +177,17 @@ export const getStaticProps: GetStaticProps<
 
   const { schema } = await readBlockDataFromDisk(blockMetadata);
 
+  const readmeMd = await fetchBlockAsset(blockMetadata, "README.md");
```

Comment:
`README.md` is not included in the NFT manifest because of 
https://github.com/vercel/nft/blob/42defbaff909c2fedb5f8a385fed97c4c4f4713b/src/analyze.ts#L175

Thus, we can’t read this from disk inside Lambda and have to fetch it instead.

Reply:
Fetch did not work. See comment in diff (`make-block-assets-compatible-with-vercel-lambdas.ts`)